spec/rubocop/cop/rspec/rails/minitest_assertions_spec.rb

```diff
@@ -271,6 +271,127 @@
+      RUBY
+
+      expect_correction(<<~RUBY)
+        expect(b).to be_within(0.001).of(a)
+      RUBY
+    end
+
+    it 'registers an offense when using `assert_in_delta` with ' \
+       'a custom delta' do
+      expect_offense(<<~RUBY)
+        assert_in_delta a, b, 1
+        ^^^^^^^^^^^^^^^^^^^^^^^ Use `expect(b).to be_within(1).of(a)`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        expect(b).to be_within(1).of(a)
+      RUBY
+    end
+
+    it 'registers an offense when using `assert_in_delta` with ' \
+       'a custom delta from a variable' do
+      expect_offense(<<~RUBY)
+        delta = 1
+
+        assert_in_delta a, b, delta
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `expect(b).to be_within(delta).of(a)`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        delta = 1
+
+        expect(b).to be_within(delta).of(a)
+      RUBY
+    end
+
+    it 'registers an offense when using `assert_in_delta` with ' \
+       'a custom delta and a failure message' do
+      expect_offense(<<~RUBY)
+        assert_in_delta a, b, 1, "must be within delta"
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `expect(b).to(be_within(1).of(a), "must be within delta")`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        expect(b).to(be_within(1).of(a), "must be within delta")
+      RUBY
+    end
+
+    it 'registers an offense when using `assert_in_delta` with ' \
+       'multi-line arguments' do
+      expect_offense(<<~RUBY)
+        assert_in_delta(a,
+        ^^^^^^^^^^^^^^^^^^ Use `expect(b).to be_within(1).of(a)`.
+                      b,
+                      1)
+      RUBY
+
+      expect_correction(<<~RUBY)
+        expect(b).to be_within(1).of(a)
+      RUBY
+    end
+
+    it 'registers an offense when using `assert_not_in_delta`' do
+      expect_offense(<<~RUBY)
+        assert_not_in_delta a, b
+        ^^^^^^^^^^^^^^^^^^^^^^^^ Use `expect(b).not_to be_within(0.001).of(a)`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        expect(b).not_to be_within(0.001).of(a)
+      RUBY
+    end
+
+    it 'registers an offense when using `refute_in_delta`' do
+      expect_offense(<<~RUBY)
+        refute_in_delta a, b
+        ^^^^^^^^^^^^^^^^^^^^ Use `expect(b).not_to be_within(0.001).of(a)`.
+      RUBY
+
+      expect_correction(<<~RUBY)
+        expect(b).not_to be_within(0.001).of(a)
+      RUBY
+    end
+
+    it 'does not register an offense when ' \
+       'using `expect(b).to be_within(1).of(a)`' do
+      expect_no_offenses(<<~RUBY)
+        expect(b).to be_within(1).of(a)
+      RUBY
+    end
+
+    it 'does not register an offense when ' \
+       'using `expect(b).not_to be_within(1).of(a)`' do
+      expect_no_offenses(<<~RUBY)
+        expect(b).not_to be_within(1).of(a)
+      RUBY
+    end
+
+    it 'does not register an offense when the delta is missing' do
+      expect_no_offenses(<<~RUBY)
+        assert_in_delta a, b, "whoops, we forgot about the actual delta!"
```

Comment:
If this is illegal for Minitest, I suggest removing this example altogether

Reply:
It's illegal at runtime but that doesn't mean people can't write it and then run rubocop which'll "safely" "fix" this by transforming it into RSpec which might not react the same way.

(I'm also pretty sure the cop itself will error without accounting for this, if memory serves)

Reply:
How far should we go into supporting invalid code? I’m more inclined to only support what would run in RSpec without errors.
We have a [weird test smoke test](https://github.com/rubocop/rubocop-rspec/blob/master/spec/smoke_tests/weird_rspec_spec.rb), but it would also run with no errors to my best memory.

Reply:
Generally my rule of thumb has been to account for "reasonable valid syntax" where it's not that much effort to do so, but I'm not fussed - I've pushed up a commit to invert this behaviour.